src/elastic/src/client/responses/bulk.rs

```diff
@@ -280,9 +280,9 @@ Use `iter` to iterate over individual errors without taking ownership of them:
 
 ```no_run
 # use elastic::prelude::*;
-# fn do_request() -> BulkErrorsResponse { unimplemented!() }
+# let client = SyncClientBuilder::new().build()?;
```

Comment:
These are causing the doc test issues...

Reply:
yeah, i missed the `errors_only` method on the `BulkRequestBuilder` when i was going through things originally. the examples make it look like you can just cast the `BulkRequest` return value to a `BulkErrorResponse`. i spent like thirty minutes trying to figure out where i needed to put `::<BulkErrorResponse>` to get it to return that instead of just `BulkResponse`.  searching in the docs for `BulkErrorsResponse` only returns results for the two definitions. i think i only found `errors_only` by accident when i was fixing those links and was grepping around for `BulkErrorsResponse` in the actual repo. i just thought it made more sense to have the usage be closer to the actual usage.

i can fix the doc errors tonight and at least stub out the method better so it looks closer to the actual usage if i can't get the SyncClientBuilder working as is

Reply:
Ahh makes sense

Reply:
okay, this should pass now